tests/oe/zerobase/zerobase.c

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <assert.h>
+#include <openenclave/enclave.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+/* non-public OE functions */
+extern const void* __oe_get_enclave_start_address(void);
```

Comment:
Question: in this PR, does this function calls the OE implementation directly or using the mystikos syscall path implemented above?

Reply:
This test uses out OE-compatibility library (``libopenenclave.a``), located under ``./oe``.

So you can call OE functions from a mystikos app that links with ``libopenenclave.a``. The functions are actually reimplemented to make extended syscalls.

Reply:
```C++
const void* __oe_get_enclave_start_address(void)
{
    uint64_t args[] = {0};
    return (const char*)syscall(SYS_myst_oe_get_enclave_start_address, args);
}
```

Reply:
```C++
oe_result_t oe_get_seal_key_by_policy_v2(
    oe_seal_policy_t seal_policy,
    uint8_t** key_buffer,
    size_t* key_buffer_size,
    uint8_t** key_info,
    size_t* key_info_size)
{
    uint64_t args[] = {
        (uint64_t)seal_policy,
        (uint64_t)key_buffer,
        (uint64_t)key_buffer_size,
        (uint64_t)key_info,
        (uint64_t)key_info_size,
    };
    return (oe_result_t)syscall(SYS_myst_oe_get_seal_key_by_policy_v2, args);
}
```

Reply:
So you can include the standard OE header (``<openenclave/enclave.h>``) and call OE functions as though you were in an enclave.